docs/source/basic-usage.rst

```diff
@@ -290,7 +301,9 @@ function. Your ``h2server.py`` should end up looking a like this:
         handle(sock.accept()[0])
 
 Running that in one shell, in your other shell you can run
-``hyper --h2 GET http://localhost:8080/``. That shell should hang, and you
+``hyper --h2 GET http://localhost:8080/``. For the ``curl`` use
+``curl -v --http2-prior-knowledge http://localhost:8080/`` command.
```

Comment:
Same as above, please remove / replace the usage of `hyper` with `curl` here.

Reply:
Did update it, for some reason forgot to push changes, should be good now